### PR TITLE
Modified YAMLs matter 1.3

### PR DESCRIPTION
**Pull request by gvargas-csa**
Fix issues: https://github.com/project-chip/matter-test-scripts/issues/119, https://github.com/project-chip/matter-test-scripts/issues/122, https://github.com/project-chip/matter-test-scripts/issues/125, https://github.com/project-chip/matter-test-scripts/issues/128, https://github.com/project-chip/matter-test-scripts/issues/129, https://github.com/project-chip/matter-test-scripts/issues/139, https://github.com/project-chip/matter-test-scripts/issues/140, https://github.com/project-chip/matter-test-scripts/issues/141

```
TC-ALOGIN-12.2
TC-DESC-2.1
TL-TIME-3.1
TC-CNET-4.22
TC-DRLK-2.1
TC-LCFG-2.1
TC-APPLAUNCHER-3.7.1
TC-APPLAUNCHER-3.8.1
TC-APPLAUNCHER-3.9.1
TC-CONTENTLAUNCHER-10.4
TC-CONTENTLAUNCHER-10.6
```

### CODE REVIEW

**Comment by raul-marquez-csa** [comment]
Added

**TC-LWM-1.1, TC-TCCM-1.1**
https://github.com/project-chip/matter-test-scripts/issues/133

**TC-DISHM-1.1, TC-DISHM-3.1, TC-DISHM-3.3**
https://github.com/project-chip/matter-test-scripts/issues/132

**TC-ICDM-3.4**
https://github.com/project-chip/matter-test-scripts/issues/154

File: src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml [hunk 1]
```diff
@@ -125,6 +125,11 @@ tests:
           "Step 4: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to PIXIT.CNET.THREAD_1ST_OPERATIONALDATASET and Breadcrumb
           field set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml:132]: In response NetworkIndex has to be added: 

 response:
          values:
              - name: "NetworkingStatus"
                value: 0
             - name: "NetworkIndex"
               value: 0

File: src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml [hunk 2]
```diff
@@ -264,6 +269,11 @@ tests:
           "Step 14: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to extended PAN ID of
           PIXIT.CNET.THREAD_1ST_OPERATIONALDATASET and Breadcrumb field set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml:276]: In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

File: src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml [hunk 3]
```diff
@@ -125,6 +125,11 @@ tests:
           "Step 4: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID and Breadcrumb field
           set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml:132]: In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

File: src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml [hunk 4]
```diff
@@ -306,6 +311,11 @@ tests:
           "Step 15: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID and Breadcrumb field
           set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml:318]: In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

File: src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml [hunk 5]
```diff
@@ -119,6 +119,11 @@ tests:
           "Step 4: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID and Breadcrumb field
           set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml:126]: In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

File: src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml [hunk 6]
```diff
@@ -266,6 +271,11 @@ tests:
           "Step 14: TH sends RemoveNetwork Command to the DUT with NetworkID
           field set to PIXIT.CNET.WIFI_1ST_ACCESSPOINT_SSID and Breadcrumb field
           set to 1"
+      verification: |
+          Verify that DUT sends NetworkConfigResponse to command with the following fields:
+          NetworkingStatus is success
+
+          NetworkIndex is 'Userwifi_netidx'
```
Comment (KishokG) [inline src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml:278]: In response NetworkIndex has to be added:

response:
values:
- name: "NetworkingStatus"
value: 0
- name: "NetworkIndex"
value: 0

File: src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml (hunk 1 above)
Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml:132]: Added

File: src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml (hunk 2 above)
Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_10.yaml:278]: Added

File: src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml (hunk 3 above)
Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml:132]: Added

File: src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml (hunk 4 above)
Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_11.yaml:320]: Added

File: src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml (hunk 5 above)
Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml:126]: Added

File: src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml (hunk 6 above)
Comment (raul-marquez-csa) [reply src/app/tests/suites/certification/Test_TC_CNET_4_9.yaml:280]: Added

**Review by raju-apple — APPROVED** [review]

**Pull request merged**